uml/state_diagrams/power_management.plantuml

```diff
@@ -0,0 +1,41 @@
+@startuml
+
+skinparam shadowing false
+
+title Power Management State Diagram
+
+state "Power Modes" as Pm {
+    state "Full Power" as Fp : entry: Set Display Brightness to Stored State\nentry: Set Bluetooth to Stored State\nentry: Set Health Notification to Stored State\ndo: Display Activated Display View
+    state "Low Power" as Lp : entry: Trigger Motor Vibration for 2 secs\nentry: Set Display Brightness State = Low\nentry: Disable Bluetooth\nentry: Disable Health Notification\ndo: Turn off display
+
+    Fp -down-> Lp : [Battery SoC <= 10% ||\n Accelerometer Force < XN (5 secs)]
```

Comment:
We need a non-functional requirement specifying the value of X Newtons to detect motion

Reply:
seems more a functional requirement